src/components/auth.js

```diff
@@ -18,6 +19,23 @@ const Auth = () => {
         };
     };
 
+    const processMicrosoftLogin = (response, data) => {
+        if(response.status !== 'unknown') {
+            dispatch({ 
+                type: LOGIN, 
+                name: data.name,
+            });
+        };
+    };
+
+    const authHandler = (err, data) => {
+        console.log(err, data);
+        dispatch({ 
+            type: LOGIN, 
+            name: data.name,
+        });
+      };
```

Comment:
Can you use the "processMicrosoftLogin" instead of the "authHandler" so that it matches the theme with Facebook?